PWGCF/FLOW/Forward/macros/AddTaskForwardFlowRun2.C

```diff
@@ -0,0 +1,120 @@
+/**
+ * @file   FTAddMyTask.C
+ * @author Freja Thoresen <freja.thoresen@cern.ch>
+ * 
+ * @brief  Add Q-cummulant forward task to train 
+ * 
+ * 
+ * @ingroup pwglf_forward_scripts_tasks
+ */
+/** 
+ * @defgroup pwglf_forward_flow Flow 
+ *
+ * Code to deal with flow 
+ *
+ * @ingroup pwglf_forward_topical
+ */
+/** 
+ * Add Flow task to train 
+ * 
+ * @ingroup pwglf_forward_flow
+ */
+AliAnalysisTaskSE* AddTaskForwardFlowRun2()
+{
+  Bool_t etagap = true;
+  Int_t mode = kRECON;
+  bool doNUA = true;
+
+
+
+  std::cout << "AddTaskForwardFlowRun2" << std::endl;
+
+  // --- Get analysis manager ----------------------------------------
+  AliAnalysisManager *mgr = AliAnalysisManager::GetAnalysisManager();
+  if (!mgr) 
+    Fatal("","No analysis manager to connect to.");
+
+  const char* name = Form("ForwardFlowQC");
+  AliForwardFlowRun2Task* task = new AliForwardFlowRun2Task(name);
+  
+  //AliForwardNUATask* task = new AliForwardNUATask(name);
+
+  TString resName = "awesome";
+
+  task->fSettings.doNUA = doNUA;
+
+
+
+  if (task->fSettings.doNUA){
+      TFile* file = new TFile("/home/thoresen/Documents/PhD/Analysis/nua.root");     
```

Comment:
There are a couple instances of hard-coded paths. Maybe it's not a problem but I thought I'd point it out.

Reply:
Hi @FrejaThoresen , the PR looks good, but indeed could you work around the hardcoded paths that @akubera  pointed out (apart form this, there's a few other mentions of your home folder)? thanks, redmer

Reply:
I'd suggest using an environment variable, say NUA_FILE, and use the [std::getenv](http://en.cppreference.com/w/cpp/utility/program/getenv) function to load the path. 
Maybe something like this:
```c++
  TString nua_filepath = std::getenv("NUA_FILE");
  if (!nua_filepath) {
    nua_filepath = "/home/thoresen/Documents/PhD/Analysis/nua.root"
    std::cerr << "Environment variable 'NUA_FILE' not found (this should be a path to nua.root).\n";
    std::cerr << "   Using default value: '" << nua_filepath << "'\n";
  }
  TFile *file = new TFile(nua_filepath);
```
Then you can use an `alien://path/to/nua.root` that can be used on the grid.

Reply:
Thanks for the comments, I forgot about the local path. I will change it to a more generic setting.